### ISSUE TITLE
Anyone else having trouble running snapshot tests in Xcode 16?

### ISSUE THREAD

**Issue by danilorequena** [ashfurrow/Nimble-Snapshots#284]
![Xcode 2024-09-19 09 24 05](https://github.com/user-attachments/assets/a0895e9c-a5e4-42d7-b7a3-c1e520e81eab)
![Xcode 2024-09-19 09 24 57](https://github.com/user-attachments/assets/748bad51-0986-43c2-b77f-838b4fa52ee4)

**Comment by ashfurrow** [comment]
What version of the library are you using? `9.6.1` is the latest. And Nimble?

**Comment by alessdiimperio** [comment]
yes, Seem to have issues with Expression not being able to be infered between Nimble and Foundation

**Comment by brainbicycle** [comment]
ran into this but already fixed in 9.8.0! Thanks all! Hope you are well Ash!
